### PR TITLE
fixed circular import crap.

### PR DESCRIPTION
**Pull request by thewarsawpakt**
I fixed the circular imports and obsoleted `__init__.py` in the `libtwirl` root, then added the framework for supporting a new root to install packages too.

### CODE REVIEW

**Review by caughtquick — APPROVED** [review]
Looks good sorry for the mistake

File: libtwirl/__main__.py [hunk 1]
```diff
@@ -22,8 +24,24 @@
 	raise IncorrectArgumentFormatError(f"Unknown operation '{args[0]}'. Valid operations are {', '.join(operations.keys())}.")
 if not len(args) > 1:
 	raise IncorrectArgumentFormatError("No packages specified.")
+try:
+	handle = Handle(args[3], "/var/lib/pacman")
+except:
```
Comment (Sophon96) [inline libtwirl/__main__.py:29]: Might want to raise a warning, like `bad directory` or similar.

Comment (thewarsawpakt) [reply libtwirl/__main__.py]: This should be fixed in the latest commit.

File: libtwirl/core/install.py [hunk 2]
```diff
@@ -32,6 +30,6 @@ def pkg_install(pkgs, options = None):
 	[formatted_targets.append(f"{pkg.db.name}/{pkg.name}-{pkg.version}") for pkg in targets]
 	print('\n'.join(formatted_targets), "\n")
 	# Install the packages
-	t = transaction.transaction_init()
+	t = transaction.transaction_init(handle)
 	[t.add_pkg(pkg) for pkg in targets]
-	transaction.transaction_commit(t)
+	transaction.transaction_commit(t)
```
Comment (Sophon96) [inline libtwirl/core/install.py:35]: What is the difference?

File: libtwirl/__main__.py (hunk 1 above)
Comment (thewarsawpakt) [reply libtwirl/__main__.py]: Except that the formatting got all messed up.

**Review by Sophon96 — APPROVED** [review]
LGTM

**Comment by thewarsawpakt** [comment]
@caughtquick Your old review became stale, by the way.

**Review by caughtquick — APPROVED** [review]
Reapproved, thanks @thewarsawpakt

**Pull request merged**